benchmarks/web-serving/faban_client/bootstrap.sh

```diff
@@ -14,7 +14,8 @@ WEB_SERVER_IP=$1
 LOAD_SCALE=${2:-7}
 
 while [ "$(curl -sSI ${WEB_SERVER_IP}:8080 | grep 'HTTP/1.1' | awk '{print $2}')" != "200" ]; do
-  sleep 1
+    echo "Cannot connect to web_server container at: ${WEB_SERVER_IP}:8080"
```

Comment:
It's not necessary that this message is displayed when we cannot connect to the web_server. We also get this when we are able to connect but the HTTP response is something other than 200.  Maybe we can write the message accordingly.

Reply:
We can say something like "Bad status code. Retrying to connect to the server..."